main_train_psnr.py

```diff
@@ -30,15 +33,45 @@
 # --------------------------------------------
 '''
 
-
 def main(json_path='options/train_msrresnet_psnr.json'):
-
+    
+    '''
+    # ----------------------------------------
+    # Step--0 (get clearml dataset)
+    # ----------------------------------------
+    '''
+    clearml_project="Super Resolution"
+    clearml_name="Eastern Township Super-Resolution RGB C"
+    dataset_location='/home/horoma/data/70.SuperResolution/ClearMLDataset/'
+    if not os.path.exists(dataset_location+'train'): 
+
+           dataset = Dataset.get(
+           dataset_id=None,  
+           dataset_project=clearml_project,
+           dataset_name=clearml_name,
+           only_completed=True
+           )
```

Comment:
add the auto download like in 
https://github.com/horomaAI/ns-agriculture/blob/main/data/data_factory.py#L34

use the path from the KAIR config for the clearml cache

Reply:
I found a problem with the procedure in ns_agriculture. It checks against a dataset stored in Clear ML cache. The way I want the super resolution training program to work is to use a dataset stored on the shared disk instead of the local drive of the Magog machines which has limited space. This is why the check is done on the final dataset. 

Thanks 

Jean

Reply:
As discussed see
https://github.com/horomaAI/ns-agriculture/blob/main/data/data_factory.py#L34-35

Reply:
`/home/horoma/data/vezinaj/clearml_cache`